codec-haproxy/src/test/java/io/netty/handler/codec/haproxy/HaProxyMessageEncoderTest.java

```diff
@@ -254,13 +254,13 @@ public void testTLVEncodeProxy() {
         assertEquals(alpnTlv.typeByteValue(), tlv.readByte());
         short bufLength = tlv.readShort();
         assertEquals(helloWorld.array().length, bufLength);
-        assertEquals(helloWorld, tlv.readBytes(bufLength));
+        assertEquals(helloWorld, tlv.readSlice(bufLength));
```

Comment:
`readBytes(...)` creates a new `ByteBuf` which needs to be released. `readSlice` returns a ByteBuf that shares the same reference count as the original buffer and so not need to be released